src/product/index.njk

```diff
@@ -1,7 +1,9 @@
 ---
-title: Team-Based Low-Code Development
 layout: layouts/page.njk
+title: Team-Based Low-Code Development
```

Comment:
@Hasmin-AC @MarianRaphael Can we do better here?

Reply:
What is this page exactly? I don't see it linked from any where

Reply:
Let's remove it?

Reply:
yes I think we should.